File: Oxygene/Cooper/Android/FirstAndroidApp/com.remobjects.FirstAndroidApp/DisplayMessageActivity.pas
namespace com.remobjects.firstandroidapp;

interface

uses
  java.util,
  android.app,
  android.content,
  android.os,
  android.util,
  android.view,
  android.widget;

type
  DisplayMessageActivity = public class(Activity)
  private
  protected
  public
    method onCreate(savedInstanceState: Bundle); override;
  end;

implementation

method DisplayMessageActivity.onCreate(savedInstanceState: Bundle);
begin
  inherited;
  var intent: Intent := getIntent();
  var message: String := intent.getStringExtra(MainActivity.EXTRA_MESSAGE);

  //  Create the text view
  var textView: TextView := new TextView(self);
  textView.TextSize := 40;
  textView.Text := message;

  //  Set the text view as the activity layout
  ContentView := textView;
end;

end.

File: Hydrogene/Cooper/Android/FirstAndroidApp/org.me.firstandroidapp/DisplayMessageActivity.cs
using java.util;
using android.app;
using android.content;
using android.os;
using android.util;
using android.view;
using android.widget;

namespace org.me.firstandroidapp
{
    public class DisplayMessageActivity : Activity
    {
		
		public override void onCreate(Bundle savedInstanceState)
		{
			base.onCreate(savedInstanceState);
			
      Intent intent = getIntent();
      string message = intent.getStringExtra(MainActivity.EXTRA_MESSAGE);

      TextView textView = new TextView(this);
      textView.TextSize = 40;
      textView.Text = message;
      ContentView = textView;
		}

    }
}
